MapboxCoreNavigation/RouteOptions.swift

```diff
@@ -3,13 +3,24 @@ import MapboxDirections
 extension RouteOptions {
     internal var activityType: CLActivityType {
         switch self.profileIdentifier {
-        case MBDirectionsProfileIdentifier.cycling, MBDirectionsProfileIdentifier.walking:
+        case DirectionsProfileIdentifier.cycling, DirectionsProfileIdentifier.walking:
             return .fitness
         default:
             return .automotiveNavigation
         }
     }
     
+    convenience init(options: RouteOptions) {
```

Comment:
Since this copy constructor lives outside of MapboxGeocoder.swift, it’ll be prone to bitrot. Fortunately, we’ll be able to remove it once mapbox/MapboxDirections.swift#388 lands.

Reply:
Indeed, it’s missing several DirectionsOptions properties required for routing, like `includesSteps`.